samples/python/video_v4l2.py

```diff
@@ -30,11 +30,11 @@ def decode_fourcc(v):
     color = (0, 255, 0)
 
     cap = cv.VideoCapture(0)
-    cap.set(cv.CAP_PROP_AUTOFOCUS, False)  # Known bug: https://github.com/opencv/opencv/pull/5474
+    cap.set(cv.CAP_PROP_AUTOFOCUS, 0)  # Known bug: https://github.com/opencv/opencv/pull/5474
 
     cv.namedWindow("Video")
 
-    convert_rgb = True
+    convert_rgb = 1
```

Comment:
It makes sense to replace passing of `convert_rgb` argument instead of changing its type:

`convert_rgb` => `1 if convert_rgb else 0`